visualization/src/pcl_visualizer.cpp

```diff
@@ -1035,6 +1041,8 @@ int
 getDefaultScalarInterpolationForDataSet (vtkDataSet* data)
 {
   vtkPolyData* polyData = vtkPolyData::SafeDownCast (data);
+  if (!polyData)
+    return (false);
```

Comment:
Maybe it's okay to leave it as is, with segfault possibility? It's a helper function anyway, should not be called from user code.